content/docs/languages-sdks/python/_index.md

```diff
@@ -149,9 +136,42 @@ $ venv/bin/pip install -r requirements.txt
 
 {{< /chooser >}}
 
+#### Poetry
+
+To add a new dependency when using `poetry`, run the `poetry add` command in your project directory:
+
+{{< chooser os "macos,windows,linux" >}}
+
+{{% choosable os macos %}}
+
+```bash
+$ poetry add ${PACKAGE_NAME}
```

Comment:
I don't have a concrete package name to use here, are we ok with using syntax like `${PACKAGE_NAME}` to indicate that this is a place holder?

Reply:
This is okay, but for command-line args, we typically use this notation, with angle-brackets and hyphens:

```
poetry add <package-name>
```

So I'd suggest sticking with that if possible.

Reply:
I think this is fine, yeah, although we don't have a particular standard around this yet (and probably should). I'll poll the team and see what they think, but IMO this is good for now.